src/com/oltpbenchmark/DBWorkload.java

```diff
@@ -264,7 +266,8 @@ public static void main(String[] args) throws Exception {
             if (xmlConfig.containsKey("numDBConnections")) {
                 wrkld.setNumDBConnections(xmlConfig.getInt("numDBConnections"));
             } else {
-                wrkld.setNumDBConnections(numWarehouses);
+                // We use a max of 200 connections per node so as to not overwhelm the DB cluster.
+                wrkld.setNumDBConnections(min(numWarehouses, wrkld.getNodes().size() * 200));
```

Comment:
We should make this a parameter, say something like `maxConnectionsPerNode`, so that it becomes easy to change this on the fly. Similar for `connectionTimeoutMillis` below.

Reply:
This is already something that is controlled using the xml config.
I don't think people would need to change it often and hence can reside in the xml.
Similarly connectionTimeoutMillis can be an xml config.

Reply:
I was referring to making the 200 value a param in the following expression: 
```
wrkld.setNumDBConnections(min(numWarehouses, wrkld.getNodes().size() * 200));
```

Worried that we would not be able to ask anyone to change this if needed. Should we introduce the param as `maxConnectionsPerTserver` so that its clear what it does, and set the default value to 200?

What do you think @ndeodhar ?

Reply:
Agree, the max should be configurable instead of being hardcoded.

Reply:
Sync'd offline with Sudheer. Since we already have a param to control `numDBConnections`, we don't need another one for `maxConnectionsPerNode`